src/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react'
+import AddCard from './AddCard'
+// import AddCardType from './AddCardType'
+import CardList from './CardList'
+
+export type FunnelStep =
+  | {
+      step: 'add'
+    }
+  | { step: 'select' }
+  | { step: 'list' }
```

Comment:
```suggestion
type Step = 'add' | 'select' | 'list'

export type FunnelStep = { step: Step }
```

step의 종류만 별도로 관리해도 괜찮을 것 같아요! 😄